docs/BR.md

```diff
@@ -749,6 +751,8 @@ The Random Value SHALL remain valid for use in a confirming response for no more
 
 **Note**: Once the FQDN has been validated using this method, the CA MAY also issue Certificates for other FQDNs that end with all the Domain Labels of the validated FQDN. This method is suitable for validating Wildcard Domain Names.
 
+Effective November 1, 2024, validations using this method MUST NOT rely on WHOIS to identify Domain Contact information.
```

Comment:
The language needs to state whether reuse of previous validation is allowed or not. Otherwise, it is unclear.

Reply:
"validations" strongly suggests "validation" events, not "issuance" events ;)

Reply:
This seems to intersect with comments from @shaver (thank you!) and @dougbeattie (thank you!), responding here as its the latest.

**From:** Effective November 1, 2024, validations using this method MUST NOT rely on WHOIS to identify Domain Contact information.

**To:** Effective November 1, 2024, Subscriber Certificates MUST NOT be issued that rely on WHOIS to identify Domain Contact information. Prior validations using WHOIS to identify Domain Contact information MUST NOT be used to issue new Subscriber Certificates, regardless of permitted data reuse periods.

Thoughts? (would be happy to see this less verbose, but also in a way that's not subject to misinterpretation)

Reply:
I don't think you need the "regardless" clause, because MUST NOT is absolute. But will this be clear enough that it affects "renewals" as well as issuance for new domains?

Reply:
@ryancdickson This does indeed clarify that it's based on "at issuance" rather than "at DCV".

@shaver "renewal" of a certificate is considered an "issuance" event, which would invoke this prohibition

Reply:
Just a minor comment regarding the effective dates. Have in mind that we need at least 7 days for discussion when the ballot is ready, another 7 for voting and then 1 month for IPR review. It´s almost impossible to have the version published by that date and be effective.

Reply:
@barrini  - Today + 44 days = October 30, 2024?

Reply:
If everything goes smooth, there are no discussions/changes, the voting is perfect, there´re no weekends when need to release the versions, etc. yes, you´re right but I think it´s too tight.

Reply:
+up to 3 business days for the Chair to publish the IPR "all clear" and final version of the document(s). To avoid a repeat of the confusion highlighted during the [discussions of SC-46](https://lists.cabforum.org/pipermail/servercert-wg/2021-May/002733.html), an immediate effective date (i.e., don't specify any effective date) would be clearer.

Reply:
Thanks for the additional context, @CBonnell.

To confirm my understanding: If I were to change...

**From:** "Effective November 1, 2024, Subscriber Certificates MUST NOT be issued that 1) rely on WHOIS to identify Domain Contact information or 2) reuse validation data where WHOIS was used to identify the Domain Contact."

**To:** "Subscriber Certificates MUST NOT be issued that 1) rely on WHOIS to identify Domain Contact information or 2) reuse validation data where WHOIS was used to identify the Domain Contact."

While also...
- updating the Ballot preamble to call out the new requirements would be effective immediately upon publication.
- and, relying on the SCWG Chair to set the effective date during publication.

That'd be an acceptable approach from your perspective?

Reply:
If the desire is to propose a ballot with a November 1st (or thereabouts) effective date, the steps you outlined are how I would recommend writing the ballot. Doing so avoids any SC46-esque confusion about effective dates in the event the BR is published on (changes come into effect midnight UTC) or after the stated effective date.

Reply:
Correct, this way, when the new version is published, in the effectiveness column of the new version is set, usually, the date of the publication, so once is published is effective. But, if you want to set a specific date and start counting from there for statistics for example, or just to know which CAs are still using that method, then set a date with enough time, let´s say Dec 1st.